ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml

```diff
@@ -10,12 +10,12 @@ ref:
   - name: EO_SUB_PACKAGE
     documentation: The package name of the Elasticsearch operator to install.
     default: elasticsearch-operator
-  - name: EO_SUB_SOURCE
+  - name: SOURCE
```

Comment:
How about keep the channel/source independently  in cases of we have such requirement in future.

Reply:
I can do that, but we need to update the code in logging web console testing too.

Reply:
Done.